### ISSUE TITLE
[Bug]: grade script clobbers manifestPlaceholder settings

### ISSUE THREAD

**Issue by dtarnawsky** [OneSignal/OneSignal-Cordova-SDK#928]
### What happened?

The [build-extras-onesignal.gradle](https://github.com/OneSignal/OneSignal-Cordova-SDK/blob/95b735cd52070f7a8c8b4d7b8b18e8b916f23cb9/build-extras-onesignal.gradle#L17) script is replacing entries in `manifestPlaceHolders` of `build.gradle`.

This means that if you have a entry in `manifestPlaceholders` called `AUTH_URL_SCHEME` you will get a manifestMerger error:
`Manifest merger failed : Attribute data@scheme at AndroidManifest.xml requires a placeholder substitution but no value for <AUTH_URL_SCHEME> is provided.`

### Steps to reproduce?

```Markdown
1. Use the repo https://github.com/dtarnawsky/onesignal-807.git
2. npx ionic build && npx cap sync
3. npx cap run android

After build you get the error: 
Error: Attribute data@scheme at AndroidManifest.xml requires a placeholder substitution but no value for <AUTH_URL_SCHEME> is provided.
```

Note: If you remove onesignal-cordova-plugin then the application will run.

### What did you expect to happen?

The gradle script should not alter any entries in manifestPlaceholders of build.gradle

### OneSignal Cordova SDK version

onesignal-cordova-plugin 5.0.1

### Which platform(s) are affected?

- [ ] iOS
- [X] Android

### Relevant log output

```Shell
> Task :app:processDebugMainManifest FAILED
        /.../android/app/src/main/AndroidManifest.xml Error:
        Attribute data@scheme at AndroidManifest.xml requires a placeholder substitution but no value for
        <AUTH_URL_SCHEME> is provided.
        
        See https://developer.android.com/r/studio-ui/build/manifest-merger for more information about the manifest
        merger.
        
        
        FAILURE: Build failed with an exception.
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by jennantilla** [comment]
@dtarnawsky thank you for reaching out! We're looking into this and will get back to you as soon as possible.

**Comment by christablogs** [comment]
Any update on this?  Ionic and Onesignal are two pretty significant platforms, so I would expect paid products on both of them would integrate well together without breaking the app.

**Comment by jbw716** [comment]
Just wanted to chime in to say that we are also experiencing issues with a paid, enterprise application as a result of this.
Thanks for any updates!

**Comment by christablogs** [comment]
Any update. It's been a month so our only option is to move to another Push notification service where it will not break our app

**Comment by emawby** [comment]
I apologize for the delay we are investigating the issue

**Comment by jennantilla** [comment]
Hello everyone! A fix for this is included in our latest release, [5.0.3](https://github.com/OneSignal/OneSignal-Cordova-SDK/releases/tag/5.0.3). Please let us know if you have any additional questions or concerns!
